bedrock/firefox/templates/firefox/features/pdf-editor-fr.html

```diff
@@ -0,0 +1,105 @@
+  </li>
+
+  <li>
+    <p>Le document apparaît dans un onglet de votre navigateur. La barre d'outils PDF apparaît en haut à droite de l'écran.</p>
+    <p><figure class="c-article-figure">
+      {{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-ouvrir.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-ouvrir-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </figure></p>
+  </li>
+
+  <li><p>Pour ajouter du texte, cliquez sur l'icône Texte et cliquez directement sur votre formulaire, là où vous souhaitez renseigner des informations.</p>
+    <p><figure class="c-article-figure">
+      {{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-texte.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-texte-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </figure></p>
+  </li>
+
+  <li><p>Vous pouvez déplacer vos blocs de texte et ajuster la taille, la police ou la couleur du texte ajouté, comme bon vous semble. Vous avez saisi trop d’informations ? Une icône Corbeille apparaît sous chaque bloc pour vous permettre de les effacer facilement.</p></li>
+
+  <li><p>Pour signer votre PDF, cliquez sur l'icône Dessin qui vous permettra d’ apposer vos initiales ou ajouter votre signature. Vous pouvez également surligner ou ajouter des images à votre document.</p>
+    <p><figure class="c-article-figure">
+      {{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-surligner.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-surligner-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </figure></p>
+  </li>
+
+  <li><p>Une fois vos modifications terminées, enregistrez le nouveau document où vous le souhaitez sur votre ordinateur au format PDF, tout simplement en cliquant sur l’icône Dossier. Donnez un nouveau nom à votre fichier et hop le voilà complet, ajouté, et classé&nbsp;!</p>
+    <p><figure class="c-article-figure">
+      {{ resp_img(
+        url="img/firefox/features/edit-pdf/fr-edit-pdf-sauvegarder.png",
+        srcset={
+            "img/firefox/features/edit-pdf/fr-edit-pdf-sauvegarder-high-res.png": "1.5x"
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </figure></p>
+  </li>
+</ol>
+
+<p>Modifier des PDF oui, mais de façon simple et sécurisée !</p>
+
+<hr>
+
+<h2>Vous n'avez pas encore Firefox ?</h2>
+<p><a href="{{ url('firefox.features.index') }}">En savoir plus</a> sur nos fonctionnalités, ou <a href="">Téléchargez Firefox</a> dès maintenant.</p>
```

Comment:
**Issue (blocking):** The second anchor tag in this line is missing an HREF that should lead to the Download Firefox page? But also, should said link only show up on non-firefox browsers? We already have a Download Firefox CTA for non-firefox browsers:

![Screenshot 2024-10-09 at 10 53 59 AM](https://github.com/user-attachments/assets/e89c5619-cd55-491b-9340-042816f386d7)

Reply:
✅